.github/workflows/docker.yaml

```
@@ -13,17 +13,33 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        docker:
-        - docker-image-14
-        - docker-image-13
-        - docker-image-12
+        pgversion:
+          - 12
+          - 13
+          - 14
+        tsversion:
+          - 2.6.0
     steps:
       - uses: actions/checkout@v2
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: build docker image
-        run: make ${{ matrix.docker }}
-        env:
-          DOCKER_BUILDKIT: 1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          build-args: |
+            PG_VER=pg${{ matrix.pgversion }}
+            TIMESCALEDB_VER=${{ matrix.tsversion }}
+          context: .
+          push: true
+          tags: ghcr.io/timescale/dev_promscale_extension:pg${{matrix.pgversion}}-latest
```

`latest` is reserved for last release usually
```suggestion
          tags: ghcr.io/timescale/dev_promscale_extension:pg${{matrix.pgversion}}-main
```

---

Yes. Latest is definitely wrong. I suspect that I should actually be naming it according to the PR/branch name or something. I was wanting to discuss this with you because you have some logic for this in your draft PR here: https://github.com/timescale/promscale_extension/pull/86